### PR TITLE
Feature- Only Members 

### PR DESCRIPTION
**Pull request by AleksandarIvezic**
### In this project we : 
 - Created app for creating posts
 - Only signed members can see the author and the posts, and create the posts
 - Other users can only read posts

### CODE REVIEW

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑🛑**

Hello @ShinobiWarior 
This is great work. I like the setup and overall code structure.
However, let's make small changes and get the project approved👍

[REQUIRED]: When I try to sign out I get the error below. Kindly fix it.
deviseerror

That's it👍

Kindly make the required changes and ask for another review👌👌

Happy coding 💻💻

**Comment by Henryhaulka** [comment]
> ### **STATUS: CHANGES REQUESTED🛑🛑🛑**
> Hello @ShinobiWarior
> This is great work. I like the setup and overall code structure.
> However, let's make small changes and get the project approved👍
> 
> [REQUIRED]: When I try to sign out I get the error below. Kindly fix it.
> deviseerror
> 
> That's it👍
> 
> Kindly make the required changes and ask for another review👌👌
> 
> Happy coding 💻💻


It's  running without error on my local pc and my partner's

So we don't  have an issue here
Screenshot (23)

File: config/routes.rb
Comment (carlos-ssh) [inline config/routes.rb]: Kindly add corresponding resources in your routes file which allows [:new, :create, :index].

**Review by carlos-ssh — APPROVED** [review]
**STATUS APPROVED** 🍾🍾👏🏽💪🏽
Hello team! 👋🏽👋🏽👋🏽

BEFORE TO MERGE:
- Kindly add corresponding resources in your routes file which allows [:new, :create, :index].

_HAPPY CODING_
@CARLOS-SSH

**Pull request merged**
